target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -606,24 +602,25 @@ ramips_setup_macs()
 	hootoo,ht-tm02|\
 	loewe,wmdr-143n|\
 	unbranded,a5-v11)
-		lan_mac=$(cat /sys/class/net/eth0/address)
```

Comment:
This clause should be safe to delete entirely, shouldn't it?

Reply:
@blocktrron 
you have to keep the case, as the default case will set up WAN MAC = LAN_MAC + 1
But you can obviously delete the line setting lan_mac.
That's why I made it an empty case in my patch 3 here.

I personally do not consider the default case very helpful here, since it will make many device support patch authors fall into it by just overlooking (similar to the discussion with the ralink_* size variables). However, I could think of no way to change this without having to check every single device in the target or by introducing a huge non-default case for 150 to 200 devices.

Reply:
All 3 devices only have 1 ethernet port, thus setting the WAN mac is kind of pointless.

Reply:
oh... and I made a patch dropping it...

Reply:
Well, it will be a dead additional entry in uci config then, if the config_generate script isn't even smart enough to not produce it then. So it shouldn't break anything.
Still, IMHO keeping the empty case would be the proper solution. Other one-port devices now falling into default case by accident or newly added might use it in the future, too.

Reply:
And the really proper solution would be to remove the default-case eventually ...